client/src/constants/publisher_key.js

```diff
@@ -1,5 +1,5 @@
-const STRIPE_PUBLISHABLE = process.env.NODE_ENV === 'production'
-  ? 'pk_live_MY_PUBLISHABLE_KEY'
-  : 'pk_test_7mDiIF0ntVgLJjKz6B7evL0i00J88bDjzD';
+//const STRIPE_PUBLISHABLE = process.env.NODE_ENV === 'production'
```

Comment:
Anything with ENV needs to be added to the Netlify.toml file so Netlify knows about it otherwise it wont know what to do with it. Same as on Heroku.